### PR TITLE
build badge component

### PR DESCRIPTION
**Pull request by Rockyraven**
create badge component

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
All looks good, can merge

File: component/badge/badge.css [hunk 1]
```diff
@@ -1,65 +1,38 @@
...
-    display: flex;
-    gap: 2rem;
-    
-}
- 
-.badge-icon{
-    display: flex;
-    gap: 4rem;
-    justify-content: space-evenly;
-    /* text-align: center; */
```
Comment (stutisk) [inline component/badge/badge.css:28]: remove these comments

File: component/badge/badge.css [hunk 2]
```diff
@@ -1,65 +1,38 @@
...
-    /* align-items: center; */
-    
-}
-h5{
+ h5{
     border: 1px  solid;
     border-radius: 50%;
-    font-size: 15px;
```
Comment (stutisk) [inline component/badge/badge.css:35]: above 10 px give it in rem

File: component/badge/badge.css [hunk 3]
```diff
@@ -1,65 +1,38 @@
...
-    padding: 1px 0 1px 3px;
-    width: 12px;
-    margin-left: 24px;
+    font-size: 1rem;
```
Comment (stutisk) [inline component/badge/badge.css:13]: make use of variables for font size .

File: component/badge/badge.css [hunk 4]
```diff
@@ -1,65 +1,38 @@
...
-}
- 
+ }
 .badge{
     margin-top: 4rem;
-    /* margin-left: 2rem; */
 }
 .symbol3{
     margin-left: 1.7rem;
-    background-color: gray;
```
Comment (stutisk) [inline component/badge/badge.css:58]: use variables for colors too!

File: component/badge/badge.css [hunk 5]
```diff
@@ -1,65 +1,38 @@
...
 }
 .symbol4{
     margin-left: 2.1rem;
-    background-color: lawngreen;
```
Comment (stutisk) [inline component/badge/badge.css:62]: same use variables

**Review by Shubhamkr2610 — APPROVED** [review]
looks goood to me.

**Review by sagargse — COMMENTED** [review]
use variables for colors otherwise all good

**Pull request merged**